File: app/src/main/java/com/fthdgn/books/api/GoogleBooksApi.java
package com.fthdgn.books.api;

import android.support.annotation.IntRange;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import com.fthdgn.books.api.response.Volumes;

public interface GoogleBooksApi {

    @GET("mylibrary/bookshelves/7/volumes")
    Call<Volumes> getMyVolumes(@Query("startIndex") @IntRange(from = 0) int startIndex);

}

File: app/src/main/java/com/fthdgn/books/api/GoogleBooksApi.kt
package com.fthdgn.books.api

import android.support.annotation.IntRange
import com.fthdgn.books.api.response.Volumes
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface GoogleBooksApi {

    @GET("mylibrary/bookshelves/7/volumes")
    fun getMyVolumes(@Query("startIndex") @IntRange(from = 0) startIndex: Int): Call<Volumes>

}
